doc/src/manual/performance-tips.md

```diff
@@ -477,6 +477,12 @@ Here, we happened to know that the first element of `a` would be an [`Int32`](@r
 an annotation like this has the added benefit that it will raise a run-time error if the
 value is not of the expected type, potentially catching certain bugs earlier.
 
+A slightly more generic way to annotate the type of `x` is `x = convert(Int32,a[1])::Int32`.  The
```

Comment:
This paragraph is supposed to make two points: First, to increase genericity, a `convert` function can be used rather than a direct annotation.  In other words, the use of `convert` allows more possible types for `a[1]`.  The second point (which is the point that surprised me) is that if `convert` is used in this manner, it needs a type annotation else it is type-unstable.